srcpkgs/flashfocus/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'flashfocus'
+pkgname=flashfocus
+version=2.0.3
+revision=1
+archs=noarch
+build_style=python3-module
+pycompile_module="flashfocus"
+hostmakedepends="python3-setuptools"
+depends="i3ipc-python python3-click python3-marshmallow
+ python3-yaml"
+short_desc="Simple focus animations for i3wm and sway"
+maintainer="travankor <travankor@tuta.io>"
+license="MIT"
+homepage="https://github.com/fennerm/flashfocus"
+distfiles="https://github.com/fennerm/flashfocus/archive/v${version}.tar.gz"
+checksum=231b260c6e18cc3cc6400d22797e5c8441432dc2c08154fdccc77c78934ba5d1
+
+pre_build() {
+	#use i3ipc instead of xcb
+	sed -i '22d;24,25d' setup.py
```

Comment:
Please write out what to remove instead from where to make sure this will work in future versions.

Reply:
or make a proper patch